### DISCUSSION TITLE
Stack dump call in svunit_testcase::give_up()

### DISCUSSION THREAD

**Discussion by edotesla** [svunit/svunit#238] [Ideas]
Note: I have an old version of the package.  Here's what I have added:

task svunit_testcase::give_up();
  event never;

`ifdef VCS
  $stack;
`endif
`ifdef NCV
  $stacktrace;
`endif
// FIXME: add other simulators

  @(never);
endtask

**Comment by tudortimi** [comment]
Seems related to #206.

It's maybe also related to #210, because then users could add a `$stacktrace` themselves.

I'm thinking about whether it might not be best to just print a stack trace from `$ERROR` and add the option to suppress it from the command line.

**Comment by edotesla** [comment]
Yes, great minds think alike!
